### ISSUE TITLE
Screenshot to jpg

### ISSUE THREAD

**Issue by gak** [gak/gravit#34]

**Comment by FrMo** [comment]
Could be done sinilar to my PNG screenshot "backend". 

Please keep in mind that the JPG format is not very good in handling "line art" images like the ones that gravit creates; the results could become a bit blurred, loose color contrast or show block artifacts.

**Comment by gak** [comment]
Yeah I understand JPEG is lossy. I just had this on the old website so I just moved them all into here. PNG is definitely a better option for screenshots.

**Comment by FrMo** [comment]
btw, JPEG may still be an option, when we disable "chroma color subsampling".
Just tried a few examples, and the results are very close to the original PNG screenshot.
